### PR TITLE
PR to modify Protobuf library as well as reintroduce workload logs.

### PR DESCRIPTION
**Pull request by AmartC**
This PR introduces changes to the payload protobuf definition to include additional fields relevant for workload logs. This PR also enables the ingest plugin to send over workload logs to the AI services through Nats.

### CODE REVIEW

File: AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java [hunk 1]
```diff
@@ -273,19 +274,38 @@ else if (ingestDocument.hasField("COMM")){
...
+        }
+        if (ingestDocument.hasField("deployment")) {
+            deployment = ingestDocument.getFieldValue("deployment", String.class);
+        }
+        if (ingestDocument.hasField("service")) {
+            service = ingestDocument.getFieldValue("service", String.class);
+        }    
         ingestDocument.setFieldValue("log_type", logType);
         ingestDocument.setFieldValue("kubernetes_component", kubernetesComponent);
+        ingestDocument.setFieldValue("pod_name", podName);
```
Comment (tybalex) [inline AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:287]: should we set those values for workload logs only?

Comment (AmartC) [reply AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:287]: We do have Longhorn and Rancher logs which do have pods and deployment data so might be worthwhile to keep those for anything we do down the road.

Comment (tybalex) [reply AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:287]: longhorn and rancher logs have their own pertained models

Comment (AmartC) [reply AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:287]: They do but if we want to do further analysis down the road where for Rancher and Longhorn log anomaly detection, we want to correlate it with a particular pod and deployment, then it'd be good to have that data available.

Comment (tybalex) [reply AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:287]: OK. do we need to set those fields as empty string if not exist?

Comment (tybalex) [reply AIOps-OpniPreProcessor/src/main/java/org/opensearch/opnipreprocessing/plugin/OpniPreProcessor.java:287]: and is it necessary to get  `deployment` and `service` and set the same values back?

**Review by tybalex — APPROVED** [review]
Look good.

**Pull request merged**